tests/unit/freelist.c

```diff
@@ -39,7 +39,7 @@ int deregmr_simple(void *handle)
 }
 
 struct random_freelisted_item {
-	int random;
+	size_t random;
```

Comment:
nack; the choice of `int` was intentional here, to make sure that we didn't have crazy alignment requirements on reginfo.